lib/robots/dor_repo/accession/end_accession.rb

```diff
@@ -39,14 +39,12 @@ def start_captioning
             # user asked for OCR but the object is not OCRable
             raise 'Object cannot be OCRd' unless ocr.possible?
 
-            # start OCR workflow
-            workflow_service.create_workflow_by_name(druid, 'ocrWF', version: next_version, lane_id:)
+            workflow_service.create_workflow_by_name(druid, 'ocrWF', version: next_version, lane_id:, context: workflow.context)
           elsif stt.required?
             # user asked for SpeechToText but the object is not speech-to-textable
             raise 'Object cannot have speech-to-text applied' unless stt.possible?
 
-            # start speechToText workflow
-            workflow_service.create_workflow_by_name(druid, 'speechToTextWF', version: next_version, lane_id:)
+            workflow_service.create_workflow_by_name(druid, 'speechToTextWF', version: next_version, lane_id:, context: workflow.context)
```

Comment:
the actual fix is adding the `context: workflow.context` to the call when creating the new workflows